xls/examples/BUILD

```diff
@@ -562,6 +563,13 @@ place_and_route(
     placement_density = "0.8",
     synthesized_rtl = ":find_index_5000ps_model_unit_verilog_synth",
     target_die_utilization_percentage = "30",
+    density_fill_config = "fill.json"
+)
+
+gds_write(
```

Comment:
should we add more rules for other examples?

Reply:
can we replace `_asic` with `_gds` or `_layout`?

Reply:
can we rename the rule?
`find_index_gds`
or 
`find_index_layout`

Reply:
I missed that somehow, It's now fixed